var/spack/repos/builtin/packages/py-netcdf4/package.py

```diff
@@ -32,6 +32,13 @@ class PyNetcdf4(PythonPackage):
     # following patch disables the usage of pkg-config at all.
     patch('disable_pkgconf.patch')
 
+    # Older versions of the package get a false negative result when checking
+    # the version of HDF5.
+    patch('check_hdf5version.patch', when='@:1.2.9 ^hdf5@1.10:')
+
+    # We can skip the 'build' phase to avoid recompilation of the library.
+    phases = ['install']
```

Comment:
I've noticed this as well, I'm thinking about switching the default `phases` to just install, but want to wait until we have better CI to try something like that.